modules/net-xlb/ssl_certificates.tf

```diff
@@ -15,26 +15,38 @@
  */
 
 locals {
-  managed = (
+  # If at least one group backend service without
+  # HC is defined, create also a default HC
+  ssl_certificates_config = (
+    try(var.target_proxy_https_config.ssl_certificates, null) == null
+    || length(coalesce(try(var.target_proxy_https_config.ssl_certificates, null), [])) == 0
+    ? merge(
+      { default = var.ssl_certificates_config_defaults },
+      coalesce(var.ssl_certificates_config, {})
+    )
+    : coalesce(var.ssl_certificates_config, {})
+  )
```

Comment:
This feels unnecessarily complicated. For example `length(coalesce(try(var.target_proxy_https_config.ssl_certificates, null), [])) == length(try(var.target_proxy_https_config.ssl_certificates, []))`.

And I think you can probably simplify it a bit considering both parts of the ternary operator include `coalesce(var.ssl_certificates_config, {})`. Perhaps something like 
```
ssl_certificates_config = merge(coalesce(var.ssl_certificates_config, {}), {
  default = var.ssl_certificates_config_defaults
})
```